.github/workflows/frozen.yml

```diff
@@ -41,3 +41,27 @@ jobs:
       if: startsWith(matrix.os, 'windows') == false
       working-directory: build
       run: cmake --build . --target test
+  build_meson:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
+    steps:
+    - name : Fetch meson
+      run : sudo apt -y install meson
```

Comment:
Probably want to pip install here, because Ubuntu has an ancient version that doesn't support c++20 and the other OSes don't use apt anyway.